spec/spec_helper.rb

```diff
@@ -94,6 +94,13 @@ def hash_inspect(hash)
   end
 end
 
+RSpec.shared_context "with warn_about_false_positives set to false", :warn_about_false_positives do
+  original_value = RSpec::Expectations.configuration.warn_about_false_positives?
+
+  before(:context) { RSpec::Expectations.configuration.warn_about_false_positives = false }
+  after(:context)  { RSpec::Expectations.configuration.warn_about_false_positives = original_value }
+end
```

Comment:
As mentioned above, I would expect this to sandbox the change rather than actually making the change...